lite/types.go

```diff
@@ -4,6 +4,9 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
+// NOTE: The Verifier interface is deprecated.  BaseVerifier can continue to
```

Comment:
Can we get rid of it? I don't think there are downstream dependencies.

Reply:
Yes